internals/overlord/servstate/handlers.go

```diff
@@ -825,6 +828,37 @@ func (s *serviceData) checkFailed(action plan.ServiceAction) {
 	}
 }
 
+// writeMetric writes the service's metrics.
+func (d *serviceData) writeMetric(writer metrics.Writer) error {
+	err := writer.Write(metrics.Metric{
+		Name:       "pebble_service_start_count",
+		Type:       metrics.TypeCounterInt,
+		ValueInt64: d.startCount.Load(),
+		Comment:    "Number of times the service has started",
+		Labels:     []metrics.Label{metrics.NewLabel("service", d.config.Name)},
+	})
+	if err != nil {
+		return err
+	}
+
+	active := 0
```

Comment:
It won't affect the metrics, but it would be nice to move the `pebble_service_active` metric to the top of `writeMetric` so that it's in the same position as the `pebble_check_up` gauge (top). It feels odd that this status gauge is last but the checks one is first. I think putting them both first is best.